contracts/src/protocol/OmniAVS.sol

```diff
@@ -214,6 +219,11 @@ contract OmniAVS is IOmniAVS, IOmniAVSAdmin, IServiceManager, OwnableUpgradeable
         return strategies;
     }
 
+    /// @inheritdoc IServiceManager
+    function avsDirectory() external view returns (address) {
```

Comment:
we should just be able to use the public getter for this. the interface wrapper is just a solidity convenience, on chain it is just the address

Reply:
Yes but solidity complains

Reply:
Complains that we have a different return type than the `IServiceManager`